package.json

```diff
@@ -67,7 +67,7 @@
     "catch-uncommitted": "ts-node --transpile-only automation/run.ts catch-uncommitted",
     "ci": "npm run test && npm run catch-uncommitted",
     "watch": "gulp watch",
-    "lint": "balena-lint -e ts -e js --typescript --fix automation/ lib/ typings/ tests/ gulpfile.js",
+    "lint": "balena-lint -e ts -e js --typescript --fix automation/ lib/ typings/ tests/ bin/balena* gulpfile.js",
```

Comment:
@srlowe, I am contributing a commit that adds `bin/balena*` to the list of `lint`ed paths. I am happy to approve the PR, but please review my review before I approve it. :-)

Reply:
Had to replace `bin/balena*` with `bin/balena bin/balena-dev` because Windows didn't understand the wildcard.